backend/src/services/super-admin/super-admin-service.ts

```diff
@@ -113,7 +121,27 @@ export const superAdminServiceFactory = ({
         });
       }
     }
-    const updatedServerCfg = await serverCfgDAL.updateById(ADMIN_CONFIG_DB_UUID, data);
+
+    const encryptWithRoot = await kmsService.encryptWithRootKey();
```

Comment:
I thought we had a org level KMS? Why don't we use that

Reply:
because the slack configuration values are instance level. They are outside the scope of any org